### PR TITLE
Run eslint on PRs

### LINKED ISSUE #19: Set up github action to trigger eslint on PRs

### PR DESCRIPTION
**Pull request by sangwoo108**
Fix https://github.com/brave/playlist-component/issues/19

You can see what it looks like when eslint fails on PR at https://github.com/brave/playlist-component/pull/20

### CODE REVIEW

**Review by simonhong — APPROVED** [review]
++ for me bot I would pass final review to other es ninjas :)

File: .github/workflows/eslint.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Install packages
+        run: |
+          npm install
+          npm install @microsoft/eslint-formatter-sarif@2.1.7
+
+      - name: Run ESLint
+        run: npx eslint .
+          --config .eslintrc.js
+          --ext .js,.jsx,.ts,.tsx
+          --format @microsoft/eslint-formatter-sarif
```
Comment (fallaciousreasoning) [inline .github/workflows/eslint.yml:32]: Wow, this is awesome @sangwoo108 - I'm going to test adding this to Leo

**Review by fallaciousreasoning — APPROVED** [review]
LGTM, the output is stellar! I wonder how hard it would be to setup in Brave-Core too

**Comment by sangwoo108** [comment]
Yeah, I didn't expect the output to be this great :)

**Pull request merged**